webui/react/src/pages/TaskList.tsx

```diff
@@ -48,6 +48,9 @@ const TaskList: React.FC = () => {
     .map(src => src.data || [])
     .reduce((acc, cur) => [ ...acc, ...cur ], [])
     .map(commandToTask);
+  if (loadedTasks.length > 0) {
+    loadedTasks[0].startTime = '1979-01-01T00:00:12.123123-06:00';
```

Comment:
non-blocking: Consider making this a constant this somewhere else.

Reply:
oops forgot to take this out. This was just an overriding change to see what duration column would look like at its longest time (41 years)